CHANGES.md

```diff
@@ -1,3 +1,11 @@
+## 1.3.0
+
+*2021-08-05*
+
+### Changed
+
+- Change the way ID packets are represented
```

Comment:
What about adding this is about PGP?  It might not be obvious for someone reading the changelog.